bliss/models/detection_encoder.py

```diff
@@ -44,6 +45,7 @@ def __init__(
         annotate_probs: bool = False,
         slack: float = 1.0,
         optimizer_params: Optional[dict] = None,
+        scheduler_params: Optional[dict] = None,
```

Comment:
What do you use the scheduler for?

Reply:
I've been using it to drop the learning rate from 1e-3 to 1e-4 after 10 or so epochs

Reply:
I'm also really interested in trying out the [OneCycleLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html) scheduler.